### PR TITLE
LPS-117248 Fixes error when trying to edit a FieldSet object field

### PR DESCRIPTION
**Pull request by matuzalemsteles**
This PR fixes some bugs found along the way that were not described in the ticket but that was possible to see before fixes them.

- Fixed error when editing the FieldSet using the custom object Sidebar
- Fixed error when trying to drag a custom object be it FieldSet or edited Field Group to the builder
- Fixed error when dragging or double click a FieldSet it becomes a Field Group

Fixes errors both when dragging and double-clicking, the implementations are different places.

### CODE REVIEW

**Comment by matuzalemsteles** [comment]
ci:test:sf

**Comment by matuzalemsteles** [comment]
cc @interaminense

**Comment by matuzalemsteles** [comment]
ci:test:relevant

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/CustomObjectFieldsList.es.js [hunk 1]
```diff
@@ -132,13 +130,21 @@ const getFieldTypes = ({
...
+					collapsible: customProperties.collapsible,
+					repeatable,
+					showLabel,
+				},
 				useFieldName: name,
 			}),
-			label,
+			label:
+				label[editingLanguageId] ??
+				label[Liferay.ThemeDisplay.getDefaultLanguageId()],
```
Comment (interaminense) [inline modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/CustomObjectFieldsList.es.js:147]: @matuzalemsteles podemos usar o defautlLanguageId do `dataDefinition.defaultLanguageId`? É uma regra do dataDefinition para a tradução de um objeto, e o defaultLanguageId nesse momento sempre irá existir.

Comment (matuzalemsteles) [reply modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/CustomObjectFieldsList.es.js:147]: Hmm, então `dataDefinition.defaultLanguageId` as vezes vem vazio é isso? Eu mantive o mesmo comportamento que estava antes para não quebrar nada, mas podemos mudar se você achar que é melhor.

Comment (interaminense) [reply modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/CustomObjectFieldsList.es.js:147]: Pode manter dessa forma, lembrei que a gente tem uma estoria de traduzir o fieldset e podemos implementar isso nesse momento @matuzalemsteles

**Comment by interaminense** [comment]
LGTM

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/CustomObjectFieldsList.es.js (hunk 1 above)
Comment (matuzalemsteles) [reply modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/CustomObjectFieldsList.es.js:147]: Beleza

**Comment by matuzalemsteles** [comment]
ci:forward

**Comment by matuzalemsteles** [comment]
ci:forward

**Comment by alexcandidos** [comment]
Hi @matuzalemsteles , I believe we can forward it directly to BChan based on the results we saw from three days ago https://github.com/matuzalemsteles/liferay-portal/pull/51#issuecomment-663790717. 

Hey @kenjiheigel @yichenroy look, we had Passed results for stable and relevant, but the PR was not forwarded to BChan, maybe it was due to some load issue on CI last Friday.

**Comment by yichenroy** [comment]
Hi @alexcandidos , the `passing` result is actually misleading because there was actually a problem with finding a downstream build, the result was actually a failure (this is a bug in our result generation, see ticket below):
```
[beanshell] java.util.concurrent.TimeoutException: Unable to find downstream build. Invocation URL:https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)
```
[Full console log](https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/6809/console)

This issue is currently under investigation: https://issues.liferay.com/browse/LRCI-1431

**Pull request closed without merging**